mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule SimpleAuth.Mixfile do
 
   defp deps do
     [
-      {:phoenix, "~> 1.3.0 or ~> 1.4.0 or ~> 1.5.0"},
+      {:phoenix, "1.4.0 or ~> 1.5.0 or ~> 1.6.0"},
```

Comment:
I think there should be a `~` before `1.4.0`

Reply:
Indeed. 😅